packages/table/stories/table-virtualized.stories.ts

```diff
@@ -97,7 +97,6 @@ class VirtualTable extends SpectrumElement {
     protected override render(): TemplateResult {
         return html`
             <sp-table
-                aria-rowcount="50"
```

Comment:
Nice catch to remove this here! Were you able to see the tests fail with this removed before you added new code to `Table.ts`?

Reply:
Yes, I observed that tests failed when it was removed, and they were repaired when my code added it back programmatically.